BossMod/Framework/ActionManagerEx.cs

```diff
@@ -210,6 +210,12 @@ public float GCD()
             return gcd->Total - gcd->Elapsed;
         }
 
+        public float GCDTime()
```

Comment:
What is the intent? If you're trying to get GCD duration - that won't work in downtime.

Reply:
That was the intent, yeah. I guess it returns 0 during downtime. I hadn't noticed any issues since all the GCD-length-dependent calculation in the rotation is dependent on being in combat.